src/Management/test/TracingBase.Test/TracingBaseServiceCollectionExtensionsTest.cs

```diff
@@ -72,5 +72,25 @@ public void AddDistributedTracing_WiresIncludedExporters()
             Assert.NotNull(serviceProvider.GetService<IOptions<ZipkinExporterOptions>>());
             Assert.NotNull(serviceProvider.GetService<IOptions<JaegerExporterOptions>>());
         }
+
+        [Fact]
+        public void AddDistributedTracing_ConfiguresSamplers()
+        {
+            // test AlwaysOn
+            var services = new ServiceCollection().AddSingleton(GetConfiguration(new Dictionary<string, string> { { "Management:Tracing:AlwaysSample", "true" } }));
+            var serviceProvider = services.AddDistributedTracing(null).BuildServiceProvider();
+            var hst = serviceProvider.GetService<IHostedService>();
+            Assert.NotNull(hst);
+            var tracerProvider = serviceProvider.GetService<TracerProvider>();
```

Comment:
Without the code change, this test would throw `System.InvalidOperationException : No service for type 'OpenTelemetry.Trace.AlwaysOffSampler' has been registered.`